web/book/src/project/contributing/development.md

```diff
@@ -451,21 +453,28 @@ Currently we release in a semi-automated way:
    require a minor release — then run
    `cargo release version $version -x && cargo release replace -x` to bump the
    version and PR the resulting commit.
+
 3. After merging, go to
    [Draft a new release](https://github.com/PRQL/prql/releases/new){{footnote: Only
-   maintainers have access to this page.}}, copy the changelog entry into the release
-   description{{footnote: Unfortunately GitHub's markdown parser
-    interprets linebreaks as newlines. I haven't found a better way of editing
-    the markdown to look reasonable than manually editing the text.}}, enter the
-   tag to be created, and hit "Publish".
+       maintainers have access to this page.}}, copy the changelog entry into the
+   release description{{footnote: Unfortunately GitHub's markdown parser
+        interprets linebreaks as newlines. I haven't found a better way of editing
+        the markdown to look reasonable than manually editing the text.}}, enter
+   the tag to be created, and hit "Publish".
```

Comment:
Some odd indents here...

Reply:
I'm pretty sure this PR addresses the odd indents that are currently in the Book (https://prql-lang.org/book/project/contributing/development.html)

The problem is that the `admonish note` cannot be indented: it gets screwed up when there are spaces at the front. But maybe I'm not seeing what you noticed

Reply:
OK cool, it does look good in HTML!